test/suite/test_timestamp04.py

```diff
@@ -109,6 +109,10 @@ def ConnectionOpen(self, cacheSize):
         self.session = wttest.WiredTigerTestCase.setUpSessionOpen(self, self.conn)
 
     def test_rollback_to_stable(self):
+
+        # Disable this test until we deal with rollback for the relevant history project
```

Comment:
Note: I have added a comment in WT-4680 to re-enable this test when we work on rollback to stable for this project.